.github/workflows/test.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         
     - name: Build package
       run: |
+        git submodule update --init --recursive
```

Comment:
@alexnguyenn  Looks like the CI is having problems fetching the submodules, thought this line would work ... do you have any ideas?